src/main/java/org/dbflute/intro/app/logic/core/FlutyFileLogic.java

```diff
@@ -44,4 +46,21 @@ public void writeFile(File textFile, String content) {
             throw new UncheckedIOException("Cannot write the text file: " + textFile, e);
         }
     }
+
+    /**
+     * Open directory by filer. (e.g. finder if mac, explorer if windows)
+     * Use OS command.
+     *
```

Comment:
add @param for the "dir" argument

Reply:
@jflute 
fixed. 
c78fbae645771dc95c0119f61f454f77102821f0